src/com/hms_networks/americas/sc/string/StringUtils.java

```diff
@@ -0,0 +1,43 @@
+package com.hms_networks.americas.sc.string;
+
+/**
+ * Class providing utilities for working with and manipulating {@link String} objects.
+ *
+ * @author HMS Networks, MU Americas Solution Center
+ * @since 1.1
+ */
+public class StringUtils {
+
+  /**
+   * Method to replace all instances of <code>before</code> with <code>after</code> in <code>string
+   * </code> and return the result.
+   *
+   * @param string string to modify
+   * @param before to replace
+   * @param after replacement
+   * @return string with replacement completed
+   * @since 1.0
+   */
+  public static String replaceAll(String string, String before, String after) {
```

Comment:
```suggestion
  public static String replace(String string, String target, String replacement) {
```

Changes to match String's replace method from 1.5
https://docs.oracle.com/javase/7/docs/api/java/lang/String.html#replace(java.lang.CharSequence,%20java.lang.CharSequence)

Reply:
This method should probably also throw the null pointer exception that the 1.5 version does if any of the strings are null.

Reply:
This should not be named replaceAll as that would imply it takes a regex, replace is more appropriate to match the forbidden 1.5 code.

Reply:
updated x2

Reply:
I think you missed the rename of before and after

Reply:
yes i did

Reply:
fixed